documentation/modules/exploit/multi/http/cve_2021_35464_forgerock_openam.md

```diff
@@ -0,0 +1,125 @@
+## Vulnerable Application
+
+### Description
+
+This module leverages a pre-authentication remote code execution vulnerability in the OpenAM identity and access
+management solution. The vulnerability arises from a Java deserialization flaw in OpenAM’s implementation of the Jato
+framework and can be triggered by a simple one-line GET or POST request to a vulnerable endpoint. Successful
+exploitation yields code execution on the target system.
+
+This vulnerability also affects the ForgeRock identity platform which is built on top of OpenAM and is thus
+is susceptible to the same issue.
+
+### Installation
+
+#### OpenAM on Ubuntu 20.04.2 x64 with Docker version 20.10.2
+
+*This maps TCP port 7080 to the containers port 8080.*
+
+1. `sudo docker run -h localhost -p 7080:8080 --name openam openidentityplatform/openam`
```

Comment:
Won't this end up pulling down the latest OpenAM vs the vulnerable one?

```
 ~/git/metasploit-framework │ add_openam_deserialization *4 ?25  sudo docker run -h localhost -p 7080:8080 --name openam openidentityplatform/openam
[sudo] password for gwillcox: 
Unable to find image 'openidentityplatform/openam:latest' locally
latest: Pulling from openidentityplatform/openam
```

Reply:
Fixed with Spencer's commit at https://github.com/rapid7/metasploit-framework/pull/15386/commits/fba838f4e89200a32f23465aacc823dd3938f91a